tests/create_grid_2d_test.py

```diff
@@ -39,6 +40,17 @@ def test_create_grid_2d_data(tmp_path):
     # Create a workspace
     workspace = Workspace(h5file_path)
 
+    grid = Grid2D.create(workspace)
+
+    with pytest.raises(AttributeError, match="The Grid2D has no geographic"):
+        grid.get_tag()
+
+    grid.u_cell_size = 20.0
+    grid.v_cell_size = 20.0
+
+    with pytest.raises(AttributeError, match="The Grid2D has no superficy"):
+        grid.get_tag()
+
     grid = Grid2D.create(
         workspace,
```

Comment:
This guy results in a malformed object in GA. Consider `remove_entity` after you are done testing.

![image](https://user-images.githubusercontent.com/55204635/204352503-cf78f288-5497-4d62-be1c-0c564a93209d.png)